### PR TITLE
cleanup deprecated stuff and simplify travis test

### LINKED ISSUE #19: Error Highlighting Shifted
In this screen shot you can see that the highlighting for the error "undeclared interface class 'UserInterface'" is over ", EquatableIn" instead of "UserInterface".

screen shot 2016-01-20 at 18 15 50

It's possible that I have configured things incorrectly. I'm just getting back into PHP have many years. However I think I've followed the instructions. I'm running PHPLint 2.1_20151116 installed via Homebrew.(https://github.com/rgant/homebrew-php/blob/up_phplint/Formula/phplint.rb Updating the formula to the latest version here. However this didn't work on 20150305 either.) Sublime Text 3 build 3083, Package Control & SublimeLinter are up to date.

Any help resolving this would be appreciated.

### PR DESCRIPTION
**Pull request by braver**
also fixes #19

### CODE REVIEW

**Comment by braver** [comment]
Good point. Lets see if I remember enough PHP to actually test this

**Comment by braver** [comment]
I decided to change this to always run on real files using tempfile_suffix `-`, because requires/imports are a really important thing in PHP (at least IIRC) and it really hampers the usefulness of this plugin if it can't follow those. 

@kaste PTAL

File: linter.py [hunk 1]
```diff
@@ -1,27 +1,8 @@
...
 
 class PHPLint(Linter):
-    """Provides an interface to the phplint executable."""
-
-    syntax = ('php', 'html')
-    cmd = 'phpl --php-version 5 --print-path relative --print-column-number --tab-size 4 --no-overall'
-    version_args = '--version'
-    version_re = r'PHPLint (?P<version>\d+\.\d+)'
-    version_requirement = '>= 2.0'
+    cmd = 'phpl --print-path relative --print-column-number --no-overall ${args}'
```
Comment (kaste) [inline linter.py]: Append $file to cmd bc we probably deprecate this automagic stuff in the future.

Comment (braver) [reply linter.py]: 👍 

Question: I see some linters declare cmd as a tuple, is there any benefit to doing it that way?

Comment (kaste) [reply linter.py]: After all we convert to a list, haha. Writing a string looks better and should be okay unless pythons shlex.split fn has a bug.

Comment (braver) [reply linter.py]: Thanks, good to know. I guess if shlex.split has bugs we’ll notice soon enough 😉

**Pull request merged**